crosswalks/ogcapi-coverages.md

```diff
@@ -0,0 +1,428 @@
+[`resample_spatial`](https://processes.openeo.org/#resample_spatial) process +
+[`POST /jobs/{jobId}/results`](https://api.openeo.org/1.2.0/#tag/Batch-Jobs/operation/start-job) (async)
+
+- OAC1: 
+  - `GET /collections/{collectionId}/coverage?scale-size=axisName({number})[,axisName({number})]*`
+  - `GET /collections/{collectionId}/coverage?scale-axes=axisName({number})[,axisName({number})]*`
+  - `GET /collections/{collectionId}/coverage?scale-factor={number}`
+
+Rescaling a dataset in **openEO** requires the same procedure as for any other kind
+of processing, thus preparing a process graph that executes the desired operation, then
+fetch the results, either synchronously or asynchronously through the creation of a
+[batch job](https://api.openeo.org/1.2.0/#tag/Data-Processing/operation/create-job).
+
+At the time of writing, the available **openEO** processes for data resampling are
+[`resample_spatial`](https://processes.openeo.org/#resample_spatial) &mdash; 
+that rescales the spatial dimensions to a target resolution, or to
+a target spatial coordinate reference system &mdash; and
+[`resample_cube_spatial`](https://processes.openeo.org/#resample_spatial)/
+[`resample_cube_temporal`](https://processes.openeo.org/#resample_cube_temporal), that
+rescale the spatial/temporal dimensions to match the resolutions of a
+provided target datacube.
+
+While **OAC1** does not let a user specify
+a resolution, it lets a client specify the target number of samples
+along a dimension, or the scaling factor. While the capabilities
+of a user-defined process with asynchronous job management
+go well beyond data the resampling of a dataset and provide 
+a wide range of benefits, **OAC1** provides a somewhat simpler shortcut
+for via query parameters of the URL.
+
+**OAC1** also allows for scaling on *any* coverage's dimension, while
+**openEO** only along the spatial plane/dimensions.
+
+
+### Tiles
+
+- openEO: `n/a`
+
+- OAC1:
+  - `GET /collections/{collectionId}/coverage/tiles` : list the available tilesets
+  - `GET /collections/{collectionId}/coverage/tiles/{tileMatrixSetId}` : tileset description
+  - `GET /collections/{collectionId}/coverage/tiles/{tileMatrixSetId}/{tileMatrix}/{tileRow}/{tileCol}` : retrieve tile
+
+**OAC1**, in alignment with the requirements defined in the
+[OGC API - Tiles](http://docs.ogc.org/DRAFTS/20-057.html) standard, allows the retrieval of
+coverage data in form of tiles, hence trimmed and resampled to match the 
+boundaries and resolution of a given tile. More than one tilesets can be attached to a coverage.
+
+**openEO** does not provide tiled access directly, but can be easily cascaded to
+a dedicated secondary tiles service (like [XYZ tiles](https://wiki.openstreetmap.org/wiki/Slippy_map_tilenames))
+thanks to the [`/service_types`](https://api.openeo.org/1.2.0/#tag/Capabilities/operation/list-service-types) resource.
+
+
+### Media encoding
```

Comment:
I'm not sure whether comparing these two are completely correct. Let's discuss later.